subjects/printifnot/README.MD

```diff
@@ -0,0 +1,45 @@
+## printifnot
+
+### Instructions
+
+Write a function that takes a `string` as an argument and returns the letter `G` if the argument length is less than 3, otherwise returns **Invalid Input\n**.
```

Comment:
- Instructions fix: "...if the argument's length is less than or equal to 3, otherwise return `Invalid Input\n`."
- With equal to 3, I mean that our solution is including  argument's with the length of 3, we need to decide if it is less than 3 exclusively, because if it is less than 3 then our solution is wrong.

Reply:
fixed